libs/ui-lib/lib/common/utils.ts

```diff
@@ -40,11 +47,17 @@ export const isStringValidJSON = (input: string): boolean => {
   }
 };
 
-export const validateFileSize = (value: string, fileSize: number): boolean => {
+export const validateFileSize = (value: string): boolean => {
   const contentFile = new Blob([value], { type: 'text/plain;charset=utf-8' });
-  return contentFile.size <= fileSize;
+  return contentFile.size <= MAX_FILE_SIZE_BYTES * MAX_FILE_SIZE_OFFSET_FACTOR;
 };
 
-export const getMaxFileSizeMessage = (fileSize: number): string => {
-  return `File size is too big. Upload a new ${fileSize / 1000} kB or less.`;
+export const getMaxFileSizeMessage = `File size is too big. Upload a new ${fileSize(
+  MAX_FILE_SIZE_BYTES,
+  0,
+  'si',
+)} or less.`;
```

Comment:
Why did you change the message? I suggested an alternative one because this one doesn't read well.

```suggestion
export const getMaxFileSizeMessage = `File size is too big. The file size must be less than ${fileSize(
  MAX_FILE_SIZE_BYTES,
  0,
  'si',
)}.`;
```